components/action-mgt/org.wso2.carbon.identity.action.execution/src/test/java/org/wso2/carbon/identity/action/execution/ActionExecutorServiceImplTest.java

```diff
@@ -133,40 +133,79 @@ public void testActionExecuteFailureWhenMultipleActionsAvailableForActionType()
         actionExecutorService.execute(ActionType.PRE_ISSUE_ACCESS_TOKEN, any(), any());
     }
 
+    @Test(expectedExceptions = ActionExecutionException.class,
+            expectedExceptionsMessageRegExp = "No action Ids found for action type: PRE_ISSUE_ACCESS_TOKEN")
+    public void testActionExecuteWithActionIdsFailureWhenNotSingleActionIdAvailable() throws Exception {
```

Comment:
This method name is not quite clear.
Isn't it 'testActionExecuteWithActionIdsFailureWhenActionIdListIsEMPTY'

Reply:
Addressed.